File: Arrays 101/26. Remove Duplicates from Sorted Array/remove_duplicates.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        bwd = 0
        for fwd in range(1, len(nums)):
            if nums[fwd] != nums[bwd]:
                bwd += 1
                nums[bwd] = nums[fwd]
        return bwd + 1

File: Arrays 101/26. Remove Duplicates from Sorted Array/remove_duplicates.cpp
class Solution
{
public:
    int removeDuplicates(vector<int> &nums)
    {
        int fwd;
        int bwd = 0;
        for (fwd = 1; fwd < nums.size(); fwd++)
        {
            if (nums[fwd] != nums[bwd])
            {
                bwd++;
                nums[bwd] = nums[fwd];
            }
        }
        return bwd + 1;
    }
};